.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -29,8 +29,9 @@ authtest () {
     $PYTHON --version
     $PYTHON -m pip install --upgrade wheel setuptools pip
     cd src
-    $PYTHON -m pip install '.[aws]'
-    $PYTHON test/auth_aws/test_auth_aws.py
+    $PYTHON -m pip install '.'
+    $PYTHON -m pip install https://github.com/blink1073/pymongo-auth-aws/archive/refs/heads/DRIVERS-2333-2.zip
```

Comment:
Can we merge https://github.com/mongodb/pymongo-auth-aws/pull/10 and update this to be pymongo-auth-aws@master?

Reply:
Done

Reply:
I still see `/refs/heads/DRIVERS-2333-2.zip`.

Reply:
Fixed